### ISSUE TITLE
Dark theme for NotABug

### ISSUE THREAD

**Issue by deep-programmer** [StylishThemes/GitHub-Dark#493]
### Feature Request

A repository of freely licensed Software, [NotABug](https://notabug.org) seems to have similar look and feel as Github web pages. Hence, I tried modifying the style locally by adding following to the top:
```CSS
@-moz-document domain("notabug.org") ...
```
But, this results in an incomplete application of the 'Github Dark' theme to it, with some parts retaining bright background i.e. being unaffected

Assuming it would require minimal changes, I would like to request for a similar dark theme for the code repository, [NotABug](https://notabug.org)

**Comment by Mottie** [comment]
Hi @deep-programmer!

It is indeed similar to GitHub's structure, but different enough that it would require someone who uses the site regularly to keep an eye on the site for any periodic changes. We also don't know if there might be conflicts between the two styles.

For now I would suggest a separate style get created. Sorry, but I won't do it since it is not a site I use regularly and my time is limited. If you can't create the style yourself, you could [request a style](https://forum.userstyles.org/categories/style-requests) in the userstyles.org forums.

**Comment by silverwind** [comment]
I've been toying with the idea of a Gogs/Gitea style, but nothing concrete yet. I think I might do it once our style generator is done, otherwise it'll be a ton of work.

**Comment by deep-programmer** [comment]
@Mottie Thanks for the suggestion. I'll make a request in userstyles.org forums. 
Also, I understand that creation of a theme for a different site ( [NotABug](https://notabug.org) ) may involve resolving conflicts and hence, may exceed effort and time requirements that can be afforded

@silverwind It would be great if you create such a style. In case you do, please reply on this thread too if possible.
Thanks for considering it.

**Comment by dllud** [comment]
I was unable to find @deep-programmer 's request at [userstyles.org forums](https://forum.userstyles.org/), thus I decided to open a request myself: [Dark theme for Gogs/Gitea (self-hosted Git services)](https://forum.userstyles.org/discussion/58576/dark-theme-for-gogs-gitea-self-hosted-git-services).

I hope @silverwind can take a look at this and perhaps make a port of the GitHub-Dark theme to Gogs/Gitea. From all the themes at userstyles.org GitHub-Dark is the best one I've ever used.

**Comment by silverwind** [comment]
I don't think I have motivation to work on this. If someone wants this, I suggest contributing a dark theme [upstream](https://github.com/go-gitea/gitea), so it can be supported and maintained there.

**Comment by dllud** [comment]
Good news. Snow_Leopard uploaded [two dark themes for Gitea into Userstyles.org](https://userstyles.org/users/494656).
